games-strategy/freeciv/freeciv-3.0.9.ebuild

```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-3 )
+
+inherit desktop lua-single qmake-utils xdg
+
+MY_PV="${PV/_beta/-beta}"
+MY_PV="${MY_PV/_rc/-RC}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="Multiplayer strategy game (Civilization Clone)"
+HOMEPAGE="https://www.freeciv.org/"
+
+if [[ ${PV} != *_beta* ]] && [[ ${PV} != *_rc* ]] ; then
+	SRC_URI="mirror://sourceforge/freeciv/${MY_P}.tar.xz"
```

Comment:
They also make Github releases it seems. Don't mind using either as long as they're identical, and one isn't missing some stuff the other is.

Reply:
we're switching to github (mostly for maintainer ease-of-use) from 3.1.0 - iirc a secondary factor was that the beta source tarballs were available from GitHub and not SourceForge.